pkg/util/deployment/deployment.go

```diff
@@ -40,6 +40,12 @@ import (
 const (
 	// The revision annotation of a deployment's replica sets which records its rollout sequence
 	RevisionAnnotation = "deployment.kubernetes.io/revision"
+	// DesiredReplicasAnnotation is the desired replicas for a deployment recorded as an annotation
+	// in its replica sets.
+	DesiredReplicasAnnotation = "deployment.kubernetes.io/desired-replicas"
+	// TotalReplicasAnnotation is the total replicas for a deployment (desired+maxSurge in case the
+	// new replica set is not satured) recorded in its replica sets.
+	TotalReplicasAnnotation = "deployment.kubernetes.io/total-replicas"
```

Comment:
This name has to have Annotation as a suffix because it is an annotation key and the Deployment prefix is redundant since the annotation is already namespaced under the deployment utils package. I don't feel strong in changing the name, the comment should make it clear what this is.